command/server/config.go

```diff
@@ -20,6 +20,7 @@ type Config struct {
 	JSONRPCAddr       string     `json:"jsonrpc_addr"`
 	Telemetry         *Telemetry `json:"telemetry"`
 	Network           *Network   `json:"network"`
+	PubSubBuffersSize uint64     `json:"pubSub_buffers_size"`
```

Comment:
Please take a look at https://github.com/0xPolygon/polygon-edge/pull/437/files#r820298161, and consider how we can rename this configuration param

Reply:
Resolving because we agreed internally these shouldn't be configurable yet